tests/e2e/hcp_cluster_test.go

```diff
@@ -0,0 +1,177 @@
+					Skip("It does't non HCP cluster - cannot run this test")
+				}
+				role := clusterDetail.AuditLogRoleARN
+				Expect(role).ToNot(BeEmpty())
+
+				By("Edit the cluster to disable audit log forwarding")
+				_, err = clusterService.EditCluster(
+					clusterID,
+					"--audit-log-arn", "",
+					"-y",
+				)
+				Expect(err).To(BeNil())
+
+				By("Get cluster description")
+				output, err = clusterService.DescribeCluster(clusterID)
+				Expect(err).To(BeNil())
+				clusterDetail, err = clusterService.ReflectClusterDescription(output)
+				Expect(err).To(BeNil())
+
+				By("Edit the cluster to enable audit log forwarding")
+				_, err = clusterService.EditCluster(
+					clusterID,
+					"--audit-log-arn", role,
+					"-y",
+				)
+				Expect(err).To(BeNil())
+
+				By("Get cluster description")
+				output, err = clusterService.DescribeCluster(clusterID)
+				Expect(err).To(BeNil())
+				clusterDetail, err = clusterService.ReflectClusterDescription(output)
+				Expect(err).To(BeNil())
+				Expect(clusterDetail.AuditLogForwarding).To(Equal("Enabled"))
+				Expect(role).To(Equal(role))
+
+			})
+
+		It("create cluster with the KMS and etcd encryption  for hypershift clusters by rosa-cli - [id:60083]",
+			labels.High, labels.NonClassicCluster,
+			func() {
+				By("Check the help message of 'rosa create cluster -h'")
+				output, err := clusterService.RetrieveHelpForClusterCreate()
+				Expect(err).To(BeNil())
+				Expect(output.String()).To(ContainSubstring("--kms-key-arn"))
+				Expect(output.String()).To(ContainSubstring("--etcd-encryption"))
+				Expect(output.String()).To(ContainSubstring("--enable-customer-managed-key"))
+
+				By("Get cluster description")
+				jsonData, err := clusterService.GetJSONClusterDescription(clusterID)
+				Expect(err).To(BeNil())
+
+				if !isHosted {
+					Skip("It does't non HCP cluster - cannot run this test")
+				}
+
+				ectdKMS := jsonData.DigString("aws", "etcd_encryption", "kms_key_arn")
+				npKMS := jsonData.DigString("aws", "kms_key_arn")
+
+				if jsonData.DigBool("aws", "etcd_encryption") {
+					Expect(ectdKMS).ToNot(BeEmpty())
+				}
+				Expect(jsonData.DigString("aws", "kms_key_arn")).To(Equal(npKMS))
+			})
+
+		It("create HCP cluster with network type can work well via rosa cli - [id:71050]",
+			labels.High, labels.NonClassicCluster,
+			func() {
+				By("Check the help message of 'rosa create cluster -h'")
+				//It is hiddened now
+				helpOutput, err := clusterService.RetrieveHelpForClusterCreate()
+				Expect(err).To(BeNil())
+				Expect(helpOutput.String()).ToNot(ContainSubstring("--no-cni"))
+
+				By("Get cluster description")
+				if !isHosted {
+					Skip("It does't non HCP cluster - cannot run this test")
+				}
+				output, err := clusterService.DescribeCluster(clusterID)
+				Expect(err).To(BeNil())
+				clusterDetail, err := clusterService.ReflectClusterDescription(output)
+				Expect(err).To(BeNil())
+				networkLine := clusterDetail.Network[0]
```

Comment:
Can you make the no-cni supported in profile first?

Reply:
Sure.I created a new card to track it.I have verified it for this type of cluster. Can I get this PR merged first?